setup.py

```diff
@@ -402,6 +402,7 @@ def finalize_options(self):
         ov_libs = [
             "libopenvino_intel_cpu_plugin.so",
             "libopenvino_intel_gpu_plugin.so",
+            "libopenvino_intel_vpux_plugin.so",
```

Comment:
Please do not add vpux plugin here as we will not be creating a manylinux package until vpux is open source

Reply:
Done